src/config.cc

```diff
@@ -449,26 +449,17 @@ void Config::initFieldCallback() {
       }},
       {"rocksdb.max_bytes_for_level_base", [this](Server* srv, const std::string &k, const std::string& v)->Status {
         if (!srv) return Status::OK();
-        if (!RocksDB.level_compaction_dynamic_level_bytes) {
```

Comment:
Copy comment from https://github.com/apache/incubator-kvrocks/pull/611#discussion_r884190774:

Why do you make these changes?

Reply:
It seems another option and with this patch you remove all usages of this config. There must be a motivation and if you remove all usages, perhaps it's necessary to remove the option also.

Reply:
> Copy comment from [#611 (comment)](https://github.com/apache/incubator-kvrocks/pull/611#discussion_r884190774):
> 
> Why do you make these changes?

After my discussion with @ShooterIT , we believe that this configuration item may not be required. If the configuration is not filled in, an error should not be returned, and the default value should be set to false.